lib/node_runner_test.go

```diff
@@ -22,7 +22,7 @@ func init() {
 	kp, _ = keypair.Random()
 }
 
-func createTestNodeRunner(n int) []*NodeRunner {
+func CreateTestNodeRunner(n int) []*NodeRunner {
```

Comment:
The old name, `createTestNodeRunner` also can be work at least in this PR.(I did not tested :))